documentation/api-key-format.md

```diff
@@ -0,0 +1,69 @@
+# API Key Format
+API Keys or MonitorApiKeys used in `dotnet monitor` are JSON Web Tokens or JWTs as defined by [RFC 7519: JSON Web Token (JWT)](https://www.rfc-editor.org/rfc/rfc7519.html).
+> **Note:** Because the API Key is a `Bearer` token, it should treated as a secret and always transmitted over `TLS` or other protected protocol.
```

Comment:
```suggestion
> **Note:** Because the API Key is a `Bearer` token, it should be treated as a secret and always transmitted over `TLS` or other protected protocol.
```